tests/common/click_testing.py

```diff
@@ -274,7 +274,7 @@ def should_strip_ansi(stream=None, color=None):
                 if value is None:
                     try:
                         del os.environ[key]
-                    except Exception:
+                    except Exception:  # nosec
```

Comment:
Note: https://bandit.readthedocs.io/en/latest/plugins/b110_try_except_pass.html#b110-test-for-a-pass-in-the-except-block
We should avoid this. Since the file is a copy paste from click I am ignoring.